README.md

```
@@ -15,37 +15,60 @@ Includes third party apps + from [awesome-brew](https://github.com/romankurnovsk
+For running Electron / building from source on Debian/Ubuntu:
+
+```sh
+sudo apt-get install build-essential procps curl file git \
+  libgtk-3-dev libnotify-dev libnss3 libxss1 libasound2
+```
 
-### Option 1
+# Install
 
-In terminal:
+### macOS — Option 1 (Homebrew cask)
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)